.gitignore

```diff
@@ -7,4 +7,6 @@ client/server
 *~
 _downloads
 .merlin
-lib/
\ No newline at end of file
+lib/
+.bsb.lock
+_esy/
```

Comment:
Did esy run in the vscode plugin folder? It shouldn't have

Reply:
maybe it was generated after some experiments. I'll remove it

Reply:
On second thoughts, let's keep it. Infact, let's drop an esy.json to develop this plugin itself. 

I wanted to make sure, however, that the plugin wasn't creating an esy.json on it's own - it's supposed to create on inside .vscode/esy